cmake/mkldnn_v1.patch

```diff
@@ -51,10 +51,10 @@ index 0bc6ea57a..4fa0a22d1 100644
 +    endif()
 +endmacro()
 +
- if(WIN32 AND ${CMAKE_CXX_COMPILER_ID} STREQUAL MSVC)
-     add_definitions(/Qpar)
-     add_definitions(/openmp)
-@@ -79,6 +115,7 @@ if (DNNL_CPU_THREADING_RUNTIME MATCHES "OMP")
+ macro(set_openmp_values_for_old_cmake)
```

Comment:
where is `set_openmp_values_for_old_cmake` used? and don't we need to check for WIN32

Reply:
@gaurides That part is a diff for the patch.  Not the diff for the file itself. `set_openmp_values_for_old_cmake` is not our code. It would be easier to look at the the file `cmake/mkldnn_v1.patch` instead of the diff. I haven't changed anything in the patch compared to the current patch. I understand it is a bit confusing.

Reply:
ah, ok. thanks for the reminder